postgresql/.fmf/version

```diff
@@ -0,0 +1 @@
+1
```

Comment:
I don't think fmf tree inside library makes sense, I am not totaly sure though, @lukaszachy ?

Reply:
I agree, got to the same conclusion. Removed now.